src/App/Search/index.js

```diff
@@ -79,6 +87,17 @@ function SearchResults({ articles, query, loading, error }) {
           }
         />
       ))}
+      {totalPages > 1 && (
+        <Pagination
```

Comment:
We should just create our own pagination component that wraps semantic's and instead works on offsets, so it would contain all that offset => page math and hide the first, last, prev, and next elements...

Reply:
Yeah, i guess.